packages/framework/ini-files/config-specs.ini

```diff
@@ -2816,6 +2816,7 @@ opt-set-cmake-var ROL_example_PDE-OPT_navier-stokes_example_01_MPI_4_DISABLE
 opt-set-cmake-var Pliris_vector_random_MPI_3_DISABLE                             BOOL : ON
 opt-set-cmake-var Pliris_vector_random_MPI_4_DISABLE                             BOOL : ON
 
+opt-set-cmake-var Zoltan_ch_7944_parmetis_parallel_DISABLE                       BOOL : ON
```

Comment:
If I am reading this INI file correctly, this disable is being set in the group:

```
[rhel8_gcc-openmpi_debug_shared_no-kokkos-arch_no-asan_complex_no-fpic_mpi_no-pt_no-rdc_no-uvm_deprecated-on_no-package-enables]
```

which is not related to ATS2, is it?

Therefore, is this test disable part of a PR that is getting a GHA build to run with ATS2?

Reply:
AT2, not ATS2 (naming is a bit confusing).  Disabling this test as part of the new autotester (2) build.  Will revisit in the future, I have a hunch as to what is going on.  That test is disabled in most of the places that we are using the new (modern) SEMS Spack-based TPL stack.